libs/moloch-v3-macro-ui/src/components/ProposalCard/ProposalCardOverview.tsx

```diff
@@ -134,7 +134,7 @@ export const ProposalCardOverview = ({
           {allowLinks && (
             <MemberCardItem asChild>
               <StyledRouterLink
-                to={`/molochv3/${daoChain}/${daoId}/members/${proposal.createdBy}`}
+                to={`/molochV3/${daoChain}/${daoId}/proposal/${proposal.proposalId}`}
```

Comment:
@jordanlesich it should be `/molochV3/${daoChain}/${daoId}/member/${proposal.createdBy}`

Reply:
Whoops. I'll add that on my next PR.